vllm/config/vllm.py

```
@@ -928,6 +918,39 @@ def has_blocked_weights():
                     # local attention.
                     self.scheduler_config.disable_hybrid_kv_cache_manager = True
 
+            if (
+                prev_disable_hma is False
+                and self.scheduler_config.disable_hybrid_kv_cache_manager is True
+            ):
+                logger.info(
```

For this code path, maybe we can raise a `NotImplementedError` with some explanation instead of a warning? Because for me if the user intentionally set `--no-disable-hybrid-manager` they are not expecting vLLM falling back to non-hybrid-allocator code path.

Explanation example:
```
Hybrid KV cache manager is explicitly enabled, but currently `--kv-events-config` is set and KV events code path is not compatible with hybrid kv cache manager.
```

---

Would `logger.warning` better fit here?